documentation/writing_yaml_plans.md

```diff
@@ -376,6 +375,35 @@ parameters:
     description: "The new application version to deploy"
 ```
 
+### Private key
+
+As a plan author, you may not want users to run your plan directly or know it exists. This is useful
+for plans that are used by other plans 'under the hood', but aren't designed to be run by a human.
+Plans accept a `private` key. The value of `private` is a boolean that tells Bolt whether to display
+the plan in `bolt plan show` or `Get-BoltPlan` output. Private plans are still viewable with `bolt
+plan show <plan_name>` and `Get-BoltPlan -Name <plan_name>`, and can still be run with Bolt.
+
+```yaml
+parameters:
+  targets:
+    type: TargetSpec
+    description: "The targets to run on"
+
+private: true
+```
```

Comment:
We should either show a complete YAML plan here (maybe using the same command step in the Puppet plan example above?) or show ellipses to indicate it's a partial plan.